pkg/products/threescale/prometheusRules.go

```diff
@@ -146,11 +152,10 @@ func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, ctx
 					},
 				},
 			},
-
 			{
 				AlertName: operatorAlertNamePrefix + "ksm-endpoint-alerts",
 				GroupName: " 3scale-operator-endpoint.rules",
-				Namespace: operatorNamespace,
+				Namespace: namespace,
```

Comment:
i'm not sure why this was like this. the prometheus rule was in the observability namespace not the observability operator namespace. that's where it should be.